src/utils/tokens.ts

```diff
@@ -6,7 +6,7 @@ import {
   TokenListContainer,
   TokenListProvider,
 } from "@solana/spl-token-registry";
-import { deserializeMint } from "@saberhq/token-utils";
+import { deserializeMint, token } from "@saberhq/token-utils";
```

Comment:
Don't believe this import is used:
```suggestion
import { deserializeMint } from "@saberhq/token-utils";
```